terraform/main.tf

```
@@ -64,7 +64,7 @@ module "repo" {
 
 resource "local_file" "github_repos" {
   content = join("\n\n", [
-    "<!-- this file is generated by Terraform -->",
+    "_This file is generated by [Terraform](github.md.tmpl)._",
```

The file gets generated under [`how_we_work/`](https://github.com/18F/tts-tech-portfolio/tree/main/how_we_work), while the template is under [`terraform/`](https://github.com/18F/tts-tech-portfolio/tree/main/terraform).

```suggestion
    "_This file is generated by [Terraform](../terraform/github.md.tmpl)._",
```